packages/editor/src/components/post-permalink/style.scss

```diff
@@ -76,7 +83,6 @@
 	// Higher specificity required to override core margin styles.
 	.editor-post-permalink-editor__save {
 		margin-left: auto;
-		flex: 0 1 auto;
```

Comment:
This is removed because it is the default and interferes with setting `flex-shrink` for all permalink buttons above.